### DISCUSSION TITLE
Can we change the mouse pos with a peace of code?

### DISCUSSION THREAD

**Discussion by SoyKiokin** [HaxeFlixel/flixel#2660] [Q&A]
Can we change the mouse pos with a peace of code? Tell me please

**Comment by Wither362** [comment]
(OLD AND NOT USEFUL)

As seen in [`FlxPointer.hx`](https://github.com/HaxeFlixel/flixel/blob/dev/flixel/input/FlxPointer.hx), you can’t change the X and Y values since they are only `(default, null)` (they can only be used by GET methods)

**Comment by Wither362** [comment]
Oh yeah, I missed:
```hx
FlxG.stage.application.window.warpMouse(0, 0);
```

**Reply by TheScratchTeen** [reply]
is `warpMouse(xx, yy)`
- Moves the mouse to X: `mouse.x + xx` | Y: `mouse.y + yy`
or
- Moves it to X: `xx` | Y: `yy` from top left of window
